test/tests/adapters/mongoose.js

```diff
@@ -135,17 +140,18 @@ describe('Adapters > SchemaAdapter', function () {
 
       var model = mongoose.model('Foo', schema);
 
-      return new SchemaAdapter(model, {
```

Comment:
Make the tests work by removing the return of the promise and keeping the done callback. Callback in case of rejected promise.